src/components/Landing.js

```diff
@@ -77,15 +53,15 @@ class Landing extends React.Component {
           <label>
             <input
               type="checkbox"
-              onChange={this.handleHideArtworkChange}
+              onChange={this.handleChange}
```

Comment:
Maybe use specific handler for this input, since it's a checkbox and needs to be handled differently?